alphadia/cli.py

```diff
@@ -341,8 +342,15 @@ def run(*args, **kwargs):
         plan.run()
 
     except Exception as e:
-        import traceback
-
-        logger.info(traceback.format_exc())
-        logger.error(e)
-        sys.exit(1)
+        if isinstance(e, CustomError):
+            message = f"An error occurred: {e.error_code}: {e.msg} {e.detail_msg}"
```

Comment:
Ideally the CLI module should have as little logic as possible so that the search behaves the same from the CLI or a jupyter notebook. I'm not sure if uncatched errors can actually reach this point or are catched before anyways.

Reply:
That's a good point .. I moved all non-CLI relevant to `planning`